app/views/application_drafts/_unavailabilities_calendar.haml

```diff
@@ -0,0 +1,41 @@
+.calendar
+  .hours_row
+    .hours.col_1
+    -Unavailability::HOURS.each do |hour|
+      .hours= hour
+  -Unavailability::DAYS.each do |dow|
+    .dow_row
+      .day.col_1= dow
+      -Unavailability::HOURS.each do |hour|
+        .hours_checkbox= form.check_box dow.downcase + '_' + hour
+
+:javascript
+  var mousedownOn = {
+    id: '',
+    checkState: false
+  };
+
+  $(document).mouseup(function() {
+    mousedownOn.id = '';
+  });
+
+  $('input[type="checkbox"]')
```

Comment:
Though if you're going to abstract into a JS file, best to narrow down this selector